app/helpers/format-date.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+import { monthNames } from 'diesel/utils/dates';
+
+export function formatDate(date) {
+  if (!date) { return; }
+
+  Ember.assert('format-date must be called with an instanceof Date',
+               date instanceof Date);
+
+  let monthName = monthNames[ date.getMonth() ];
+  return `${monthName} ${date.getDate()}, ${date.getFullYear()}`;
```

Comment:
@sandersonet @fancyremarker check out this use of ES2015 string interpolation. These are also multi-line capable.